### DISCUSSION TITLE
Error installing De-Drm tools 10.0.9

### DISCUSSION THREAD

**Discussion by PYPeke** [noDRM/DeDRM_tools#587] [Q&A]
I am sure I missed something, but I keep getting this error.  Windows 10 and Calibre 7.14.

![drm error](https://github.com/user-attachments/assets/a8446ca7-5797-447c-b23f-387a13032e7b)

Thank you for your help.

**Comment by ElleKayEm** [comment]
Unzip that file and load DeDRM_plugin.zip.

**Reply by PYPeke** [reply]
I tried to unzip several times, but it would not unzip. I finally dragged the file to my desktop and it worked. Thank you.

**Comment by sometimescasey** [comment]
I ran into this error message as well but for a different reason from you - I had downloaded and unzipped the repo, not the release. x-posting here in case it helps others: https://github.com/noDRM/DeDRM_tools/discussions/550#discussioncomment-11937419
